src/catchup/ApplyBufferedLedgersWork.cpp

```diff
@@ -86,4 +93,13 @@ ApplyBufferedLedgersWork::onRun()
 
     return State::WORK_RUNNING;
 }
+
+std::string
+ApplyBufferedLedgersWork::getStatus() const
+{
+    return mLastAppliedBufferedLedger == 0
```

Comment:
`ConditionalWork` already returns `mConditionedWork.getStatus()` if `mConditionedWork` is running . 

I made the change to use `ConditionalWork`, but there was a small issue because the status can change very frequently since we track by milliseconds. This can result in a large burst of log messages. I changed the log message in `ConditionalWork` to essentially only change every 100 milliseconds.